.github/workflows/test-docker-image.yml

```diff
@@ -14,15 +14,15 @@ jobs:
       matrix: ${{ steps.set-matrix.outputs.matrix }}
       version: ${{ steps.set-version.outputs.version }}
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v4
     - id: set-version
       run: |
-        RAKU_VERSION=$(ls |egrep '^[0-9]{4}.[0-9]{2}$')
+        RAKU_VERSION=$(ls -A -d * | egrep '^[0-9]{4}.[0-9]{2}$')
         echo "::set-output name=version::$RAKU_VERSION"
```

Comment:
Again, not in scope, but as long as you're at it `set-output` is deprecated https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Reply:
I know.... there are many workflows I've pushed, which do now complain about this... I have so far ignored this message... hoping, GitHub will somehow "auto-solve" this by themselves... because there are (ten / hundred) thousands of such workflows out there...

Once GH would really disable it, forcing customers and users to act, I'll find some time to change Star and those workflows here...